doc/source/getting_started/basics.rst

```diff
@@ -1524,10 +1524,10 @@ To iterate over the rows of a DataFrame, you can use the following methods:
 
     df
 
-iteritems
+items
 ~~~~~~~~~
```

Comment:
this needs to be same length as the header

Reply:
Ok, done.